Pod/Classes/PINAnimatedImageManager.m

```diff
@@ -41,15 +46,22 @@ + (instancetype)sharedManager;
 
 @implementation PINAnimatedImageManager
 
-+ (void)initialize
++ (void)load
```

Comment:
Wait, this is to clear images from the temporary directory? Correct me if I'm wrong but I thought the temp directory is always cleared by the system for every run of the app.